<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AC Automotores</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>PyQT 2024/Prueba/src/gui/img/AC Automotores.jpg</normaloff>PyQT 2024/Prueba/src/gui/img/AC Automotores.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#f4eee2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="11" column="2">
        <widget class="QPushButton" name="btnGuardar">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 0);</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="2">
        <widget class="QDateEdit" name="dtpFechaRegistro">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(244, 238, 226);
border: none;
border-bottom: 1px solid black;
font: 12pt &quot;Liberation Serif&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: black</string>
         </property>
         <property name="text">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLineEdit" name="txtCorreo">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(244, 238, 226);
border: none;
border-bottom: 1px solid black;
font: 12pt &quot;Liberation Serif&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QPushButton" name="btnCancelar">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 157, 255);</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item row="12" column="3">
        <widget class="QPushButton" name="btnEliminar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 0);</string>
         </property>
         <property name="text">
          <string>Eliminar</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: black</string>
         </property>
         <property name="text">
          <string>Fecha Registro</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: black</string>
         </property>
         <property name="text">
          <string>Direcci√≥n</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="2">
        <widget class="QPushButton" name="btnModificar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 0);</string>
         </property>
         <property name="text">
          <string>Modificar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="14" column="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QLineEdit" name="txtDireccion">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(244, 238, 226);
border: none;
border-bottom: 1px solid black;
font: 12pt &quot;Liberation Serif&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QLineEdit" name="txtTelefono">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(244, 238, 226);
border: none;
border-bottom: 1px solid black;
font: 12pt &quot;Liberation Serif&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="txtNombre">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(244, 238, 226);
border: none;
border-bottom: 1px solid black;
font: 12pt &quot;Liberation Serif&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Unispace</family>
           <pointsize>50</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: black</string>
         </property>
         <property name="text">
          <string>Clientes</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: black</string>
         </property>
         <property name="text">
          <string>Correo</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: black</string>
         </property>
         <property name="text">
          <string>Tel√©fono</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>88</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="0" colspan="4" alignment="Qt::AlignRight">
        <widget class="QLabel" name="lblModificar">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 170, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="3" colspan="2">
        <widget class="QCheckBox" name="checkBox">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Filtrar por fecha de Registro</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: black</string>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QPushButton" name="btnBuscar">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border-radius: 25px;</string>
         </property>
         <property name="text">
          <string>üîç</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Desde</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QDateEdit" name="dtpFechaDesde">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(244, 238, 226);
border: none;
border-bottom: 1px solid black;
font: 12pt &quot;Liberation Serif&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtBuscar">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(244, 238, 226);
border: none;
border-bottom: 1px solid black;
font: 12pt &quot;Liberation Serif&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Liberation Serif</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hasta</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QDateEdit" name="dtpFechaHasta">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(244, 238, 226);
border: none;
border-bottom: 1px solid black;
font: 12pt &quot;Liberation Serif&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QTableWidget" name="tblClientes">
         <property name="minimumSize">
          <size>
           <width>791</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Correo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tel√©fono</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Direcci√≥n</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fecha Registro</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item row="0" column="5">
          <property name="text">
           <string>6</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClientes">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Clientes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVehiculos">
    <property name="title">
     <string>Vehiculos</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVentas">
    <property name="title">
     <string>Ventas</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInicio">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Inicio</string>
    </property>
   </widget>
   <addaction name="menuInicio"/>
   <addaction name="menuClientes"/>
   <addaction name="menuVehiculos"/>
   <addaction name="menuVentas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>txtDireccion</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>txtTelefono</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>txtCorreo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>txtNombre</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>txtNombre</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>dtpFechaDesde</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1086</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>1171</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>dtpFechaHasta</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1128</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>1143</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGuardar</sender>
   <signal>clicked()</signal>
   <receiver>lblModificar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>lblModificar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
